sumpy/expansion/local.py

```diff
@@ -515,25 +521,21 @@ def m2l_translation(self) -> M2LTranslationBase:
             return self.m2l_translation_override
         else:
             from sumpy.expansion.m2l import DefaultM2LTranslationClassFactory
+
             factory = DefaultM2LTranslationClassFactory()
-            return (
-                factory.get_m2l_translation_class(self.kernel, self.__class__)())
-            from sumpy.expansion.m2l import DefaultM2LTranslationClassFactory
-            factory = DefaultM2LTranslationClassFactory()
-            return factory.get_m2l_translation_class(self.kernel,
-                self.__class__)()
+            return factory.get_m2l_translation_class(self.kernel, self.__class__)()
```

Comment:
This seems to have been a merge gone wrong? Didn't check to see where it comes from..